.github/workflows/build.yml

```
@@ -17,9 +17,9 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.8, 3.9, 3.10]
```

You just need to add in the quotations otherwise it will try to run python 3.1 
```suggestion
        python-version: ["3.8", "3.9", "3.10"]
```

---

You're a little too fast for me :)